torch_geometric/loader/temporal_dataloader.py

```diff
@@ -17,7 +18,9 @@ class TemporalDataLoader(torch.utils.data.DataLoader):
         **kwargs (optional): Additional arguments of
             :class:`torch.utils.data.DataLoader`.
     """
-    def __init__(self, data: TemporalData, batch_size: int = 1, **kwargs):
+    def __init__(self, data: TemporalData, batch_size: int = 1,
+                 device: Union[torch.device, str] = 'cpu',
+                 negative_sampling=False, neighbor_loader_size=10, **kwargs):
```

Comment:
update doc string, with `neighbor_loder_size` , and also explain how last neighbor loader is used.

Reply:
Also update tests for `TemporalDataloader` to test `negative_sampling=True`.